ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template4.kt

```diff
@@ -234,7 +234,7 @@ private fun SelectPackageButton(
 
     var columnModifier = modifier.clip(RoundedCornerShape(UIConstant.defaultCornerRadius))
 
-    val applicationContext = LocalContext.current.applicationContext.toAndroidContext()
+    val applicationContext = LocalContext.current.toAndroidContext()
```

Comment:
Any reason why we were using `Context.applicationContext` in several places?

Reply:
Thanks for explaining that! I refactored the implementation to remove the reference to `Context`. Would this approach work?